usage/Geometry/PolyDataViewer.js

```diff
@@ -1,46 +1,92 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import { View, GeometryRepresentation, PolyData } from 'react-vtk-js';
+import {
+  View,
+  GeometryRepresentation,
+  Geometry2DRepresentation,
+  PolyData,
+} from 'react-vtk-js';
+
+import { Representation } from '@kitware/vtk.js/Rendering/Core/Property/Constants';
+import { DisplayLocation } from '@kitware/vtk.js/Rendering/Core/Property2D/Constants';
+import { Coordinate } from '@kitware/vtk.js/Rendering/Core/Coordinate/Constants';
 
 // React complains about unique key prop but I don't see why
 function Example(props) {
   return (
-    <div style={{width: '100vw', height: '100vh'}}>
-      <View id="0" background={[0.1, 0.5, 0.9]}>
+    <div style={{ width: '100vw', height: '100vh' }}>
+      <View id='0' background={[0.1, 0.5, 0.9]}>
         <GeometryRepresentation
-          id="1"
+          id='1'
           property={{
-            opacity: 0.5,
+            opacity: 0.1,
             color: [0.7, 0, 0],
           }}
         >
           <PolyData
-            id="2"
+            id='2'
```

Comment:
in general we prefer html attribute to be in `"` rather than single one.

Reply:
I see. I believe that was prettier auto-fixing it. I'll revert that.